R/spatial_rt_process.R

```diff
@@ -29,13 +29,13 @@ spatial_rt_process <- function(log_state_rt,
   delta <- matrix(data = 0, nrow = n_subpopulations, ncol = n_time)
 
   # delta constr.
-  delta[, 1] <- mvrnorm(
+  delta[, 1] <- MASS::mvrnorm(
```

Comment:
Just a note, always ok to do this IMO, but since you already have #' `@importFrom MASS mvrnorm` in`R/wwinference-package.R`, you don't HAVE to